loky/backend/resource_tracker.py

```diff
@@ -75,6 +75,10 @@
 
 if os.name == "posix":
     _CLEANUP_FUNCS['semlock'] = sem_unlink
+    if sys.version_info >= (3, 8):
```

Comment:
This is leftover from #242, and became orthogonal to this PR. In the future we should support `shared_memory`, but maybe we can do it in another PR.

Reply:
+1 for removing it as it's confusing and not needed any time soon.